lib/ssh_scan/api.rb

```diff
@@ -86,11 +86,22 @@ class API < Sinatra::Base
           :timeout => 2,
           :verbosity => nil,
           :fingerprint_database => "fingerprints.db",
+          :db => "mongodb",
+          :save => "false"
         }
         options[:sockets] <<
           "#{params[:target]}:#{params[:port] ? params[:port] : "22"}"
         options[:policy_file] = options[:policy]
         options[:uuid] = SecureRandom.uuid
+        options[:db] = params[:db].downcase unless params[:db].nil?
+        options[:save] = params[:save] unless params[:save].nil?
```

Comment:
What's the vision for the "save" parameter? Shouldn't we save all results?

Also, it doesn't make sense to me why the client would specify why type of database the API should use.  Shouldn't DB type just that be transparent to the end user and it be a function of the API's configuration file?